packages/nifti-volume-loader/src/helpers/scaleNiftiArray.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Given a pixel array, rescale the pixel values using the rescale slope and
+ * intercept
+ *
+ * Todo: add the scaling of PT and SUV
+ * @param array - The array to be scaled.
+ * @param scalingParameters - The scaling parameters
+ * @returns The array being scaled
+ */
+export default function scaleNiftiArray(
```

Comment:
Could we call this modalityScaleNifti or something that explains it isn't a scaling of the image, but of the pixel values.